README.md

```diff
@@ -231,56 +231,40 @@ You can force the direction of a relationship by using:
-Person(user1, "User 1")
-Person(user2, "User 2")
-Person(user3, "User 3")
-
-System(system, "System")
-
-' normal sprites are too big 
-Rel_L(user, user2, "informs", "courier", "normal sprites are too big", $sprite="person2")
-
-' scaled sprites are ok
-Rel_R(user, user3, "informs", "courier", "scaled sprites are OK", $sprite="person2,scale=0.5")
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+HIDE_STEREOTYPE()
 
-' combine sprite and scale to a new sprite
-!$combinedSprite="person2,scale=0.5"
-Rel_R(user, user3, "informs", "courier", "combined sprites are OK", $sprite=$combinedSprite)
+Person(a, "A")
+Person(b, "B")
+Person(c, "C")
+Person(d, "D")
+Person(e, "E")
 
+BiRel_U(a, b, "talk with")
+BiRel_R(a, c, "talk with")
+BiRel_D(a, d, "talk with")
+BiRel_L(a, e, "talk with")
 
-' special smaller sprites can be used
-sprite $triangle {
-    00000000000
-    00000F00000
-    0000FBF0000
-    0000FBF0000
-    000F999F000
-    000F999F000
-    00F66666F00
-    00F66666F00
-    0F3333333F0
-    0F3333333F0
-    0FFFFFFFFF0
-    00000000000
-}
-Rel_R(user1, system, "orders", "http", "small sprites, like the small triangle", $sprite="triangle")
+Person(x, "X")
+System(s1, "S1")
+System(s2, "S2")
+System(s3, "S3")
+System(s4, "S4")
 
-' if sprite starts with &, sprite defines a OpenIconic, details see https://useiconic.com/open/
-Rel_D(user, user1, "requests", "async message", "if sprite starts with &, it defines a OpenIconic like &envelope-closed", $sprite = "&envelope-closed")
+Rel_U(x, s1, "uses")
+Rel_R(x, s2, "uses")
+Rel_D(x, s3, "uses")
+Rel_L(x, s4, "uses")
 @enduml
 ```
 
-![Relation with sprite or OpenIconic](https://www.plantuml.com/plantuml/png/bLJVQnin37w_lq8DeMiXoQN9DiWWKDPnCDRHqjBdujXAOkgpRvPlMHdxt-SVSPEikK7rujCdIzyd8TybEMPTMwlYH6gkMe6mTgwXJsLfsMQqacxTBbf2oursgDs8cxfi5DCXPqXEABaehzuFRmFqyFswh1avj1vwl0ePlzoe2TMBMxHaz5aeDO3UWpzwv_lWnHQ5YqDyal798JxD-DJZnVspPwtFA1u-almGUGVQs9efeCPAXmJC8ZXZO25NKDoXUhpUYifiKYzz1lNy9pUjbMZ3PtSL7-qdUDvhei198YRE58g35FCKAU_sAAUTb4VoRxuTOHl4Y_Fnw4FYvQPUI8tRH61Q92bUC33GkDb6YfF-zgguxwpu1hsvMBVYV_YysZ2c1haCe_NpLMXViZdJiC30AOg4GTzPoVHA8VmkmDjuPpk_ElIhpzN__6escrNTVlKnMDNbLzDaLPUVRVnAxvyysRJyBwjhh40RHniUOZZZOF9O1g3a4u9R8oGyZsH_CJAMza4kyoh4nqwmaMuDfuEC2bnAZGGCRXhKNxdHaWyywfXK18IxNuBNAcCu_WQClrt6BhxizYC-P8i_MYGNks3qh3dKICHM681EET8TbP8QFaNz4vMd779b2CMkNPX3xrNqlBX4BTfQ_GK0 "Relation with sprite or OpenIconic")
+![(unidirectional) relationship versus bidirectional relationship](https://www.plantuml.com/plantuml/png/RP11QuD044Rl_eeq9mED4lNKKneLMh1KD87s9AiEZNHTPNSaxR_lZ59KoF6zntuCUpGeD0wjj1uQLScXXiqLiJTFhgl5pVbgy3gKWm5TTGf1eLDrhTjBeVZDtc0jcz8DWttAwlAMkAqm29fK4T8BqIZGJi_xBwzHNEJdE8lVpvzfREyiAmLjEcMBnytsURlxnvBmD6D56CvO4qOp0c5CQ9sQ36HnuJ4UG26_DpUwdjgKCaxLtHHngk-cX1Eiqdpu3_aFulpN8BIsEH3dXuNmM7WBuRFm5o9W4V3cT3vDZZE30KDsEfobjvRHAFsrJ4OPpF88ggQ__mC0 "(unidirectional) relationship versus bidirectional relationship")
+
+## Layout (arrange) elements (without relationships)
```

Comment:
I like this header being added. Makes things much cleaner. 👍

Reply:
Hi @Potherca,

TOC is updated.

Related to `The README is getting really long...`: I agree ;-) Who has time? 
I think we should split C4.puml,... too. Atm, I think it is a v3.0 topic.

BR Helmut